test/promql/tests/data_plane/backup_failed_test.yaml

```diff
@@ -0,0 +1,30 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.backup_alerts.yaml
+
+tests:
+  - interval: 1m
+    input_series:
+      # No backup failures
+      - series: velero_backup_failure_total{namespace="openshift-adp", schedule="backup-toolchain-member", source_cluster="cluster01"}
```

Comment:
We have a 2nd schedule for toolchain, i.e. backup-toolchain-host. should we add it as input too?

Reply:
I don't think we need to. The values for `schedule` are only used to populate the summary/description and this is test data anyway.

Reply:
ok, thanks for explanation